system-tests/services/test-pods-communication.js

```diff
@@ -0,0 +1,77 @@
+require("../_support/utils/ServicesUtil");
```

Comment:
we wanted to group tests by feature, test-pod.js will be create, so communication will no fit into that one. we can add more tests regarding pod communication to this one later.